DiscountedPrice.py

```diff
@@ -0,0 +1,17 @@
+price = input("Enter the price: ")
+
+print("The initial price is: " + str(price))
+price = int(price)
+
+if price > 300:
+    price = price - (30/100*price)
+elif price >= 200 & price < 300:
+    price = price - (20/100*price)
+elif price >= 100 & price < 200:
+    price = price - (10/100*price)
+elif price < 100:
+    price = price - (5/100*price)
+elif price < 0:
+    price = price
+
+print("The new price is: " + str(price))
```

Comment:
You can print the price without converting to string.
`print("The new price is: " + price)`
or you can print using format option:
`print(f"The new price is: {price}")`

Reply:
Done. The 1st method is not working, because, for the if statements, I had to convert the string entered from the keyboard to an int() and then it cannot be concatenated with a str(). The second print, with format option , is ok.